Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:latest AS builder
```

Comment:
let's don't version this in the repository and make it a example (could be under docs/examples or so).

Reply:
Moved it to the tests under testdata.